JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/ViewModels/BaseViewModel.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MvvmCross.ViewModels;
+using JhipsterXamarin.Core.Resources;
+
+namespace JhipsterXamarin.ViewModels
+{
+    public abstract class BaseViewModel : MvxViewModel
```

Comment:
Split class into two files

Reply:
Remove the sncd never used class in commit 73d60d8